config/database.yml

```diff
@@ -1,30 +1,33 @@
-# PostgreSQL. Versions 8.2 and up are supported.
+# PostgreSQL. Versions 9.3 and up are supported.
 #
 # Install the pg driver:
 #   gem install pg
-# On OS X with Homebrew:
+# On macOS with Homebrew:
 #   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
-# On OS X with MacPorts:
+# On macOS with MacPorts:
 #   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
 # On Windows:
 #   gem install pg
 #       Choose the win32 build.
 #       Install PostgreSQL and put its /bin directory on your path.
 #
 # Configure Using Gemfile
-# gem 'pg'
+# gem "pg"
 #
 default: &default
   adapter: postgresql
   encoding: unicode
-  # For details on connection pooling, see rails configuration guide
-  # http://guides.rubyonrails.org/configuring.html#database-pooling
-  pool: <%= ENV['DATABASE_POOL_SIZE'] %>
-  user: <%= ENV['DB_ENV_POSTGRES_USER'] %>
-  password: <%= ENV['DB_ENV_POSTGRES_PASSWORD'] %>
-  host: <%= ENV['DB_PORT_5432_TCP_ADDR'] %>
-  port: <%= ENV['DB_PORT_5432_TCP_PORT'] %>
-  database: <%= ENV['DB_ENV_POSTGRES_DATABASE'] %>
+  pool:     <%= ENV.fetch("RAILS_MAX_THREADS", 5) %>
+  database: <%= ENV.fetch("POSTGRES_DATABASE", "feeder-db") %>
+  username: <%= ENV.fetch("POSTGRES_USER", "feeder-user") %>
+  password: <%= ENV.fetch("POSTGRES_PASSWORD", "feeder-pass") %>
+  host:     <%= ENV.fetch("POSTGRES_HOST", "127.0.0.1") %>
+  port:     <%= ENV.fetch("POSTGRES_PORT", "5432") %>
+  variables:
+    # Set a default timeout of 1 hour (for e.g. TTD flights running on the slow queue)
```

Comment:
This seems unnecessary for Feeder.

Reply:
the variables/timeout? fair enough, will delete.